### DISCUSSION TITLE
recipes/ingredients/frosting/brown-butter-cream-cheese-frosting/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#999] [Comments]
# recipes/ingredients/frosting/brown-butter-cream-cheese-frosting/

A collection of my recipes.

https://nicholaswilde.io/recipes/ingredients/frosting/brown-butter-cream-cheese-frosting/

**Comment by nicholaswilde** [comment]
Half recipe for 2 tiered cake
